### DISCUSSION TITLE
Help to display correct names from PARAM.SFO

### DISCUSSION THREAD

**Discussion by Metro-City** [aldostools/webMAN-MOD#579] [Q&A] [answered]
Hi, for "Option to display original file name or title name from PARAM.SFO."
Can you tell me what is the correct setting to see ps3 games in chronological order and to read from PARAM.SFO?
because he sees them staggered and does not read * .sfo but long or abbreviated names. Thanks

**Comment by aldostools** [comment, marked as answer]
There is no way to list games in chronological order.
The games are sorted by title (or by ISO file name or folder name if the option PARAM.SFO is unchecked).

Moreover, the sort only uses the first 6 characters to group the items.
Therefore the titles are only partially sorted. Some custom rules are applied to improve the order, but it's not always the best.

One alternative to sort your games would be renaming the ISO files or folders to begin with 3 or 4 numbers like (0001 Title, 0002 Title, etc.). That way the numbers would take precedence in the sort order. However you have to sort the files manually.

Another method is to scan the games, then customize mygames.xml with the proper order.
This method requires to disable the content scanning on startup to prevent that the XML is ovewritten.
If this is done, it is recommended to keep a backup of the XML in case that the content is re-scanned by mistake.
